com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs

```diff
@@ -2634,16 +3155,73 @@ void PrepareGPUProbeData(CommandBuffer cmd, HDCamera hdCamera, CullingResults cu
                     m_CurrentScreenSpaceShadowData[i].valid = false;
                 }
 
+                /* ---------------------------- Step 1 ---------------------------- */
+                // Do not pre-process per view...
```

Comment:
What do you mean?
note: avoid comment with /* -- */ in it, a it prevent to use /* for multiple ligne when we want to comment a large part of the code.
Prefer 
/* ---------------------------- Step 1 ---------------------------- */
+> 
// ---------------------------- Step 1 ----------------------------

Reply:
Doing anything per-view is obviously expensive. It should be avoided if the data does not depend on the eye index.

Reply:
You can change the style of comments if you like.